app/Models/Statuslabel.php

```diff
@@ -166,7 +167,22 @@ public static function getStatuslabelTypesForDB($type)
 
         return $statustype;
     }
+    public function validate()
+    {
+        $validator = Validator::make($this->attributes, $this->rules);
```

Comment:
I think if we do it this way, we're not actually checking for type though, no? We don't have that constraint in the model `$rules`:

```php
    protected $rules = [
        'name'  => 'required|string|unique_undeleted',
        'notes'   => 'string|nullable',
        'deployable' => 'required',
        'pending' => 'required',
        'archived' => 'required',
    ];
```

since `type` isn't an actual field on the table.